gallery/src/components/NxVulnerabilityCounter/NxVulnerabilityCounterPage.tsx

```diff
@@ -0,0 +1,176 @@
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTextLink, NxTable, NxP, NxCode } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxVulnerabilityCounterExample from './NxVulnerabilityCounterExample';
+import NxVulnerabilityCounterWrappingExample from './NxVulnerabilityCounterWrappingExample';
+import NxVulnerabilityCounterSmallExample from './NxVulnerabilityCounterSmallExample';
+import NxVulnerabilityCounterColumnExample from './NxVulnerabilityCounterColumnExample';
+import NxVulnerabilityCounterSmallColumnExample from './NxVulnerabilityCounterSmallColumnExample';
+import NxVulnerabilityCounterGridExample from './NxVulnerabilityCounterGridExample';
+import NxVulnerabilityCounterSmallGridExample from './NxVulnerabilityCounterSmallGridExample';
+
+const NxVulnerabilityCounterCode = require('./NxVulnerabilityCounterExample?raw'),
+    NxVulnerabilityCounterWrappingCode = require('./NxVulnerabilityCounterWrappingExample?raw'),
+    NxVulnerabilityCounterSmallCode = require('./NxVulnerabilityCounterSmallExample?raw'),
+    NxVulnerabilityCounterColumnCode = require('./NxVulnerabilityCounterColumnExample?raw'),
+    NxVulnerabilityCounterSmallColumnCode = require('./NxVulnerabilityCounterSmallColumnExample?raw'),
+    NxVulnerabilityCounterGridCode = require('./NxVulnerabilityCounterGridExample?raw'),
+    NxVulnerabilityCounterSmallGridCode = require('./NxVulnerabilityCounterSmallGridExample?raw');
+
+const NxVulnerabilityCounterPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        Component for a series of small counters which display a number representing a
+        count of vulnerabilities, and a short pre-set text string which displays
+        the severity (high, critical, medium, low, and none).
+      </NxP>
+      <NxP>
+        Each count is optional. If no value is provided for a given count, then the indicator for that severity level
+        will not be rendered.
+      </NxP>
+      <NxP>
+        Three basic layouts have been demonstrated below. It is expected that one of these should satisfy any
+        requirement.
+      </NxP>
+      <NxTable>
+        <NxTable.Head>
+          <NxTable.Row>
+            <NxTable.Cell>Prop</NxTable.Cell>
+            <NxTable.Cell>Type</NxTable.Cell>
+            <NxTable.Cell>Required</NxTable.Cell>
+            <NxTable.Cell>Details</NxTable.Cell>
+          </NxTable.Row>
+        </NxTable.Head>
+        <NxTable.Body>
+          <NxTable.Row>
+            <NxTable.Cell>criticalCount</NxTable.Cell>
+            <NxTable.Cell>number</NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell>
+              A numerical value, the number of high vulnerabilities.
```

Comment:
```suggestion
              A numerical value, the number of critical vulnerabilities.
```
I think there is a mix-up in the documentation. `criticalCount` mentions "high". and `highCount` mentions "critical".
![image](https://github.com/user-attachments/assets/feef10ce-2a4a-4fe4-86f8-2e2397dc34e9)